### PR TITLE
Remove NooBaa CSV

### PR DESCRIPTION
**Pull request by jarrpa**
- [x] Remove NooBaa CSV from OCS CSV
- [x] Maintain ownership of OB/OBC CRDs
- [x] Update make/scripts to include noobaa-operator-bundle in the catalog index 
- [x] Mark NooBaa *CRD* as dependency

Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

**Comment by umangachapagain** [comment]
@jarrpa If we're making this operator as non standalone, do we need to do these?
```
Update make/scripts to include noobaa-operator-bundle in the catalog index
Mark NooBaa CRD as dependency?
```
Since now we need to start by installing odf-operator, we should probably keep things minimal here.

**Comment by jarrpa** [comment]
This needs https://github.com/openshift/release/pull/21442 before it can pass CI.

**Comment by obnoxxx** [comment]
> This needs [openshift/release#21442](https://github.com/openshift/release/pull/21442) before it can pass CI.

Is this a chicken-and-egg thing?

File: hack/common.sh [hunk 1]
```diff
@@ -78,6 +78,15 @@ INDEX_FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${OPERATOR_INDEX_
 CATALOG_FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${CATALOG_IMAGE_NAME}:${IMAGE_TAG}"
 MUST_GATHER_FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${MUST_GATHER_IMAGE_NAME}:${IMAGE_TAG}"
 
+DEFAULT_NOOBAA_BUNDLE_REGISTRY="quay.io/dannyzaken"
```
[the remark this answers is not in the document]
Comment (jarrpa) [reply hack/common.sh:81]: :+1:

**Comment by jarrpa** [comment]
> > This needs [openshift/release#21442](https://github.com/openshift/release/pull/21442) before it can pass CI.
> 
> Is this a chicken-and-egg thing?

For once, not really! (I hope) The openshift/release PR wont break CI for other ocs-operator PRs.

File: hack/common.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (jarrpa) [reply hack/common.sh]: Done

**Review by umangachapagain — APPROVED** [review]

**Review by umangachapagain — APPROVED** [review]
/lgtm

**Pull request merged**